### PR TITLE
Blog Post

### PR DESCRIPTION
**Pull request by ak18f**

### CODE REVIEW

File: 2019-10-15-AustinK/AustinKim/_index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+education: College
+
+social:
+- icon: twitter
+  icon_pack: fab
+  link: https://twitter.com/
+- icon: github
+  icon_pack: fab
+  link: https://github.com/
```
Comment (whanley) [inline 2019-10-15-AustinK/AustinKim/_index.md:31]: add your github username

File: 2019-10-15-AustinK/AustinKim/_index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+# Display name
+name: Ak18f
```
Comment (whanley) [inline 2019-10-15-AustinK/AustinKim/_index.md]: Use your real name

File: 2019-10-15-AustinK/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Rest in Peace Miss F. Collins"
+subtitle: "Obituary"
+summary: "Emotions are running low as many, not only in Luxor but also in Cairo, are mourning the death of a local favorite, Miss F. Collins"
+authors: [AustinKim]
+tags: ["news", "passing"]
+categories: [curiosities]
+date: YYYY-MM-DD
```
Comment (whanley) [inline 2019-10-15-AustinK/index.md]: today's date

File: 2019-10-15-AustinK/index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+featured: false
+draft: false
+image:![Obituary](featured.png)
```
Comment (whanley) [inline 2019-10-15-AustinK/index.md]: only need `image:`

**Review by whanley — COMMENTED** [review]
Good use of markdown. Follow the inline comments I made.

I don't understand the relevance of the external links you use. They do not inform the reader of anything that will help them to understand the story better. Perhaps look for something on obituaries in history? Or obituaries in colonial settings?

Generally the writing is good, however.

**Comment by whanley** [comment]
Have you updated this blog post?

**Comment by ak18f** [comment]
Yeah

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**